tikv-client/src/main/java/com/pingcap/tikv/types/TimestampType.java

```diff
@@ -72,16 +72,9 @@ protected Object doConvertToTiDBType(Object value)
     if (value instanceof Long) {
       throw new ConvertNotSupportException(value.getClass().getName(), this.getClass().getName());
       // result = new java.sql.Timestamp((Long) value);
-    } else if (value instanceof String) {
-      result = java.sql.Timestamp.valueOf((String) value);
-    } else if (value instanceof java.sql.Date) {
-      result = new java.sql.Timestamp(((java.sql.Date) value).getTime());
-    } else if (value instanceof java.sql.Timestamp) {
-      result = (java.sql.Timestamp) value;
     } else {
-      throw new ConvertNotSupportException(value.getClass().getName(), this.getClass().getName());
+      return convertToMysqlDateTime(value);
```

Comment:
add some ut?

Reply:
update the documnet?
![image](https://user-images.githubusercontent.com/5574887/62760047-a1ecac00-bab5-11e9-9a33-288fbd731615.png)

Reply:
@marsishandsome logic was not changed.